compiler/qsc_frontend/src/compile.rs

```diff
@@ -516,18 +516,23 @@ fn resolve_all(
     assigner: &mut HirAssigner,
     package: &ast::Package,
     mut dropped_names: Vec<TrackedName>,
-) -> (Names, Locals, Vec<resolve::Error>) {
+) -> (
+    Names,
+    Locals,
+    Vec<resolve::Error>,
+    qsc_data_structures::namespaces::NamespaceTreeRoot,
```

Comment:
nitpick: can the errors stay at the end? It looks worse for the diff but something in my head likes the errors list to be the last or the first thing in a tuple. Not a big deal; I'm fine if you shrug and leave it as-is.